### ISSUE TITLE
Screen reader does not announce label information on emojis control in windows.

### ISSUE THREAD

**Issue by msftedad** [facebook/react#36421]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26200.8246)
Browser: Edge (Version 147.0.3912.98), Chrome (Version 148.0.7778.96)
Screen Reader: NVDA (2026.1), JAWS (2026.2603.86), Narrator
URL: https://ealush.com/emoji-picker-react/

**User Impact:**
Screen reader users cannot identify the purpose of the emoji control as its label is not announced. This makes it difficult for them to understand or use the control, impacting accessibility and task completion.

**Repro Steps:**

1. Open the URL: https://ealush.com/emoji-picker-react/ in edge/chrome browser.
2. emoji picker for React page will appear.
3. Press tab key to reach to the emoji smiley button.
4. Verify whether screen reader announces label information on emojis control in windows or not.

**Actual Result:**
Screen reader does not announce label information on emojis control in windows.
Screen reader announces as 'clickable list with 8 items grinning face button'

**Refer Attachment:**

1. Screen reader does not announce label information on emojis control in windows.png
2. Screen reader does not announce label information on emojis control in windows.mp4

https://github.com/user-attachments/assets/80b090dd-44a6-4741-9694-42ac5fbc00ae
<img width="1948" height="1182" alt="Image" src="https://github.com/user-attachments/assets/f30d071f-fea5-4faf-b2e7-774002e5e228" />

**Expected Result:**
Screen reader does not announce label information on emojis control in windows.
Screen reader should announce as 'Smileys and People clickable list with 8 items grinning face button'

**Comment by guillaumebrunerie** [comment]
@msftedad This is the wrong place to report those issues, please go to https://github.com/ealush/emoji-picker-react/issues instead.
